src/Microsoft.Management.Configuration.UnitTests/Tests/ConfigurationMixedElevationTests.cs

```diff
@@ -0,0 +1,176 @@
+    /// </summary>
+    [Collection("UnitTestCollection")]
+    [OutOfProc]
+    [OutOfProcOnly]
+    public class ConfigurationMixedElevationTests : ConfigurationProcessorTestBase
+    {
+        private readonly UnitTestFixture fixture;
+        private readonly ITestOutputHelper log;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ConfigurationMixedElevationTests"/> class.
+        /// </summary>
+        /// <param name="fixture">Unit test fixture.</param>
+        /// <param name="log">Log helper.</param>
+        public ConfigurationMixedElevationTests(UnitTestFixture fixture, ITestOutputHelper log)
+            : base(fixture, log)
+        {
+            this.fixture = fixture;
+            this.log = log;
+        }
+
+        /// <summary>
+        /// Verifies that creating a high integrity unit processor. Must run elevated.
+        /// </summary>
+        /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
+        [Fact]
+        public async Task ApplyElevatedUnit()
+        {
+            string resourceName = "E2ETestResource";
+            string moduleName = "xE2ETestResource";
+            Version version = new Version("0.0.0.1");
+
+            ConfigurationSet configurationSet = this.ConfigurationSet();
+
+            ConfigurationUnit unit = this.ConfigurationUnit();
+            unit.Metadata.Add("version", version.ToString());
+            unit.Metadata.Add("module", moduleName);
+            unit.Metadata.Add("securityContext", "elevated");
+            unit.Type = resourceName;
+            unit.Intent = ConfigurationUnitIntent.Apply;
+
+            configurationSet.Units = new ConfigurationUnit[] { unit };
+
+            IConfigurationSetProcessorFactory dynamicFactory = await this.fixture.ConfigurationStatics.CreateConfigurationSetProcessorFactoryAsync(Helpers.Constants.DynamicRuntimeHandlerIdentifier);
+
+            ConfigurationProcessor processor = this.CreateConfigurationProcessorWithDiagnostics(dynamicFactory);
+
+            ApplyConfigurationSetResult result = processor.ApplySet(configurationSet, ApplyConfigurationSetFlags.None);
+            Assert.NotNull(result);
+            Assert.Null(result.ResultCode);
+            Assert.Equal(1, result.UnitResults.Count);
+
+            foreach (var unitResult in result.UnitResults)
+            {
+                Assert.NotNull(unitResult);
+                Assert.False(unitResult.PreviouslyInDesiredState);
+                Assert.False(unitResult.RebootRequired);
+                Assert.NotNull(unitResult.ResultInformation);
+                Assert.Null(unitResult.ResultInformation.ResultCode);
+                Assert.Equal(ConfigurationUnitResultSource.None, unitResult.ResultInformation.ResultSource);
+            }
+        }
+
+        /// <summary>
+        /// Verifies the behavior of running a restricted unit.

```

Comment:
I don't know what this test is actually assessing; it seems unnecessary right now.